Resources/Prototypes/Entities/Mobs/NPCs/pets.yml

```
@@ -777,6 +777,15 @@
     name: ghost-role-information-smile-name
     description: ghost-role-information-smile-description
     rules: ghost-role-information-nonantagonist-rules
```

```suggestion
    rules: ghost-role-information-nonantagonist-rules
    raffle: null
```
You can simply add this line and remove all the other changes to avoid the reparenting.

---

oh, i thought that didnt work, trying it now to see